palm/plugins/dbt/commands/cmd_test.py

```diff
@@ -91,9 +91,18 @@ def build_test_command(
 def set_env_vars(environment, defer: bool = False) -> dict:
     plugin_config = environment.plugin_config('dbt')
     env_vars = dbt_env_vars(environment.palm.branch)
+
+    # These env vars are renamed in dbt v1.5.0, old env vars are deprecated
+    if plugin_config.is_dbt_version_greater_than("1.5.0"):
```

Comment:
Should use `or_equal` here (and in the cmd_run file), should it not?

Reply:
Yeah, it totally should be. I'll push an update to this.